docker/services/docker-compose-sidecar.yml

```diff
@@ -0,0 +1,11 @@
+services:
+  sidecar:
+    image: ${L2_DOCKER_TAG:-specularl2/sidecar:latest}
+    build:
+      context: ../..
+      dockerfile: ./services/sidecar/Dockerfile
+    volumes:
+      - ${SPECULAR_DATADIR:-$PWD/data}:/specular
+    env_file:
+      - ../../e2e/data/e2e.env
```

Comment:
does the env file path need to be relative to the context defined above?

Reply:
no, relative to the file. it throws an error if the file is not found so I'm sure it's not failing silently